src/views/communities/components/AddCommunityDialog.js

```diff
@@ -0,0 +1,343 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Box,
+  Button,
+  TextField,
+  MenuItem,
+  Select,
+  FormControl,
+  InputLabel,
+  Typography,
+  Modal,
+  Divider,
+  IconButton,
+  Avatar,
+  Grid
+} from '@mui/material';
+import PersonIcon from '@mui/icons-material/Person';
+import UploadDark from 'assets/images/icons/upload-dark';
+import UploadLight from 'assets/images/icons/upload-light';
+import { createCommunity } from 'services/createCommunity';
+import { fetchUsers } from 'services/fetchUsers';
+
+const LogoUploader = ({ logoImage, handleLogoUpload }) => (
```

Comment:
Instead of creating this logo uploader and cover uploader, create seperate component named ImageUploader, so that can be used throughout the app.